Dockerfile

```diff
@@ -2,15 +2,23 @@ FROM centos:7
 MAINTAINER      Federico Simoncelli <fsimonce@redhat.com>
 
 RUN yum update -y && \
-    yum install -y golang openscap-scanner git && \
+    yum install -y openscap-scanner git && \
     yum clean all
 
+ENV PATH=$PATH:/usr/local/go/bin
+ENV CGO_ENABLED=0
+
 COPY .  /go/src/github.com/openshift/image-inspector
 
-RUN GOBIN=/usr/bin \
+RUN cd /tmp && \
+    curl -LO https://storage.googleapis.com/golang/go1.7.linux-amd64.tar.gz && \
```

Comment:
I think you should install via yum. centos repo already has golang-1.8.3-1.el7.x86_64 available